LondonBritishLibrary/orient/BLorient5425.xml

```diff
@@ -0,0 +1,162 @@
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <idno>BL Oriental 5425</idno>
+                        <altIdentifier><idno>Or. 5425</idno></altIdentifier>
+                        <altIdentifier><idno>Strelcyn 78</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <title ref="LIT5888AsmatPrayer">Prayer against Bāryā, Legewon, and magicians</title>
+                            <incipit xml:lang="gez">በስሙ፡ ለእግዚአብሔር፡ አብ፡ በስሙ፡ ለእግዚአብሔር፡ ወልድ፡ በስሙ፡ ለእግዚአብሔር፡ መንፈስ፡ ቅዱስ፡ ተዖስ፡ አዝዮስ፡ አዛንዮስ፡ 
+                                <gap reason="ellipsis"/> በዝ፡ አስማተ፡ ቃልከ፡ ውጉዘ፡ ይኩኑ፡ አጋንንት፡ ጸዋጋን፡ ወመናፍስት፡ 
+                                <supplied reason="undefined" resp="PRS8999Strelcyn">ር</supplied>ኩሳን፡ ወባርያ፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <title ref="LIT5888AsmatPrayer">Prayer for expelling budā, Bāryā, demons, devils, and different diseases</title>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/>በስመ፡ አልፋ፡ አልፋ፡ ወኦ፡ ቤጣ፡ የውጣ፡ ሴቃ፡ ቤቃ፡ ጼቃ፡ <gap reason="ellipsis"/>
+                                ፃዕ፡ ፃዕ፡ ወፂአከ፡ ኢትግባዕ፡ ዳግመ፡ ይቤሉከ፡ አግዚአብሔር፡ ዝክረ፡ ስምከ፡ 
+                                ይደም<supplied reason="undefined" resp="PRS8999Strelcyn">ሰ</supplied>ስ፡ ለዓለመ፡ ዓለም፡ አሜን፡ ያቂ፡ ያቂ፡ ወያቂ፡ አንተ፡ ቡዳ፡ 
+                                ወአንተ፡ ባርያ፡ አንተ፡ ሳይጣን፡ ወአንተ፡ ጋኔን፡ አንተ፡ ፌራ፡ ወአንተ፡ ንዳድ፡ አንተ፡ ፈላሻ፡ ወአንተ፡ ዳክታን፡ አንተ፡ ባርያ፡ ወአንተ፡ ሌጌዎን፡ አንተ፡ ገብሮ፡ ወአንተ፡ ጢኖ፡ 
+                                አንተ፡ እኂዞ፡ ወአንተ፡ ሥራይ፡ አንተ፡ ቁር<supplied reason="undefined" resp="PRS8999Strelcyn">ፀ</supplied>ት፡ ወአንተ፡ ፍልፀት፡ 
+                                አንተ፡ መጋኛ፡ ወአንተ፡ ጉሥምት፡ አንተ፡ አቁያፃት፡ ወአንተ፡ ቁመኛ፡ ወአንተ፡ ቃግሥት፡ ወአንተ፡ ቀግስጥ፡ ዘትትሜስሉ፡ በብዙኅ፡ አምሳል፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <title ref="LIT5889UndoingOfCharms"/>
+                            <incipit xml:lang="gez">ቆቤር፡ ጅጅብ፡ ላኂቢላሆሙ፡ ለዜቤለ፡ <gap reason="ellipsis"/> ፍታሕ፡ ሥራየ፡ ዘተገብረ፡ በላዕለ፡ ገብርከ፡ መድኅን፡ 
+                                <gap reason="illegible"/> ብእሲትየ፡ ወውሉድየ፡ ወአዝማድየ፡ ወዘአኀዝኩ፡ ኵሎ፡ በእዴየ፡ ተማኅፀንኩ፡ ከመ፡ ኢይቅረቡኒ፡ ሶበድአት፡ ወቃግስት፡ 
```

Comment:
Is `<gap reason="illegible"/>` the best way of encoding "magical characters"? If there isn't a way to do it, maybe it would be good to decide on a way (also for future cases). It feels like a feature that might interest people

Reply:
I do not know another solution, but I will add a note.